apps/explorer/src/components/HomeMetrics/CurrentEpoch.tsx

```diff
@@ -5,17 +5,21 @@ import { formatDate, formatAmountParts } from '@mysten/core';
 import { format, isToday, isYesterday } from 'date-fns';
 import { useMemo } from 'react';
 
-import { NetworkStats } from './NetworkStats';
-
 import { useEpochProgress } from '~/pages/epochs/utils';
+import { Card } from '~/ui/Card';
 import { Heading } from '~/ui/Heading';
 import { ProgressBar } from '~/ui/ProgressBar';
 import { Text } from '~/ui/Text';
+import { LinkWithQuery } from '~/ui/utils/LinkWithQuery';
 
 export function CurrentEpoch() {
     const { epoch, progress, label, end, start } = useEpochProgress();
 
     const formattedDateString = useMemo(() => {
+        if (!start) {
```

Comment:
`useEpochProgress` was returning bogus data when the data from the API hadn't been retrieved yet, so previously start would equal `0` on the first render causing us to render `Wed Dec 31 1969 19:00:00 G`. I modified the hook to return `undefined` instead so we can properly guard against this case